### DISCUSSION TITLE
How to add a new model trained into TTS Coqui?

### DISCUSSION THREAD

**Discussion by Thlenzi** [coqui-ai/TTS#3535] [Qs for Live Q&A]
Hello fellows,

I'll be honest with you. 

I'm new in this field but in love for the potential of this project. 

I have used this Colab: [https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI)

I've generated a model and a now ai have this files:

- added_IVF241_Flat_nprobe_1.index
- total_fea.npy
- mi-test2600.pth
- mi-test_G_2600.pth
- mi-test_D_2600.pth

So, i'm looking for a tutorial or documentation that help to implementation the custom model into TTS python code.

If anyone can send a guide, link, tutorial or video... it's will be very appreciated 😊😊😊

Thanks in advance and have a nice year ✌️

**Comment by rosx27** [comment]
have you found a solution? i'm facing kinda similar issue, I downloaded a model from [huggingface](https://huggingface.co/myshell-ai/OpenVoice/tree/main/checkpoints/base_speakers/EN) but don't know where to put it. I've tried copying the model to the same folder as the other model downloaded via API but I'm getting an error.
